auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -527,22 +639,33 @@ static class Identity implements ModifiableRealmIdentity {
         private final Charset hashCharset;
         private final Encoding hashEncoding;
         private final SecretKey secretKey;
-
-        Identity(final String name, final Path path, final IdentityLock lock, final Charset hashCharset, final Encoding hashEncoding, Supplier<Provider[]> providers, final SecretKey secretKey) {
+        private final PrivateKey privateKey;
+        private final PublicKey publicKey;
+        private final ConcurrentHashMap<String, String> mainIndex;
+
+        Identity(final String name, final Path path, final IdentityLock lock, final Charset hashCharset,
+                final Encoding hashEncoding, Supplier<Provider[]> providers, final SecretKey secretKey,
+                final PrivateKey privateKey, final PublicKey publicKey,
+                final ConcurrentHashMap<String, String> mainIndex) {
             this.name = name;
             this.path = path;
             this.lock = lock;
             this.hashCharset = hashCharset;
             this.hashEncoding = hashEncoding;
             this.providers = providers;
             this.secretKey = secretKey;
+            this.privateKey = privateKey;
+            this.publicKey = publicKey;
+            this.mainIndex = mainIndex;
         }
 
         public Principal getRealmIdentityPrincipal() {
             return new NamePrincipal(name);
         }
 
-        public SupportLevel getCredentialAcquireSupport(final Class<? extends Credential> credentialType, final String algorithmName, final AlgorithmParameterSpec parameterSpec) throws RealmUnavailableException {
+        public SupportLevel getCredentialAcquireSupport(final Class<? extends Credential> credentialType,
+                final String algorithmName, final AlgorithmParameterSpec parameterSpec)
+                throws RealmUnavailableException {
```

Comment:
Just a minor but this file has many of newlines added to the methods params and other places which could be removed

Reply:
Yes sorry, I accidentally opened this in vscode which auto-formatted the file that I didn't notice right away. 
I'll be fixing all of these.